addons/web/static/tests/helpers/mock_server.js

```diff
@@ -528,117 +528,197 @@ var MockServer = Class.extend({
         return modelFields;
     },
     /**
-     * Simulate a call to the 'search_panel_select_range' method.
+     * Simulate a call to the search_panel_local_counters method.
+     * A model is added to the params to perform the adequate read_group.
      *
-     * Note that the implementation assumes that 'parent_id' is the field that
-     * encodes the parent relationship.
+     * @param {string} model
+     * @param {string} fieldName
+     * @param {Object} kwargs
+     * @returns {Object}
+     */
+    _mockSearchPanelLocalCounters: function (model, fieldName, kwargs) {
+        const countDomain = Domain.prototype.normalizeArray([
+            ...(kwargs.search_domain || []),
+            ...(kwargs.category_domain || []),
+            ...(kwargs.filter_domain || []),
+            ...(kwargs.group_domain  || []),
+            [fieldName, '!=', false],
+        ]);
+        const field = this.data[model].fields[fieldName];
+        let getGroupId;
+        if (field.type === 'many2one') {
+            getGroupId = group => group[fieldName][0];
+        } else {
+            getGroupId = group => group[fieldName];
```

Comment:
type === 'selection'? It should be written somewhere which field types are supported here

Reply:
it is done in the doc above